docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+-------------
+
+*Returns:* `image`
+
+
+[float]
+[[location_fn]]
+=== `location`
+
+Uses the browser's location functionality to get your current location. Usually 
+quite slow, but fairly accurate.
+
+*Accepts:* `null`
+
+*Returns:* `datatable`
+
+
+[float]
+[[l_fns]]
+== L
+[[lt_fn]]
+=== `lt`
+
+Returns whether the _context_ is less than the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[lte_fn]]
+=== `lte`
+
+Returns whether the _context_ is less than or equal to the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[m_fns]]
+== M
+[[mapColumn_fn]]
+=== `mapColumn`
+
+Adds a column calculated as the result of other columns, or not. See also <<alterColumn_fn>> and <<staticColumn_fn>>.
```

Comment:
I'm not sure what this is saying. I think I'm confused by the `or not` bit.

Reply:
So `mapColumn` will add a new column to your datasource if your provide a new column name, or it can override an existing column with new values if you provide an existing column name. if you don't provide any arguments, it won't change anything, which is what the `or not` is referring to

Reply:
Could we say:

`Adds a column calculated as the result of other columns. Changes are made only when you provide arguments.`